### PR TITLE
Migrate to go modules

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@kolyshkin ptal

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/mrunalp/fileutils
+
+go 1.15
```
Comment (kolyshkin) [inline go.mod]: I'd change that to 1.13, as this denotes a min Go version this package requires

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by mrunalp** [comment]
@thaJetzah fyi

**Comment by mrunalp** [comment]
@thaJeztah fyi

**Comment by thaJeztah** [comment]
Thanks!

**Pull request merged**